observability/readme.md

```diff
@@ -35,6 +36,12 @@ Read on how Observability can be implemented in [Machine Learning](ml-observabil
 
 As much as possible, configuration and management of observability assets such as cloud resource provisioning, monitoring alerts and dashboards must be managed as code. Observability as Code is achieved using any one of Terraform / Ansible / ARM Templates
 
+## Observability in Microservices
```

Comment:
Would recommend this be moved below the "Examples of Observability as Code" section

Reply:
I don't fully see that section there as well. I see the microservices something like the CI/CD Pipelines and Machine Learning. Thoughts?

Reply:
It's towards the end of the readme.

Reply:
@NarmathaBala I just changed the readme.md, tell me what do you think.